datadog_checks_dev/datadog_checks/dev/tooling/testing.py

```diff
@@ -311,3 +311,26 @@ def get_tox_env_python_version(env):
     match = re.match(PYTHON_MAJOR_PATTERN, env)
     if match:
         return int(match.group(1))
+
+
+def process_checks_option(checks, source=None):
+    # provide common function for determining which checks to run validations against
+    # `source` determines which method for gathering valid checks, default will use `get_valid_checks`
+
+    if source is None or source == 'checks':
+        get_valid = get_valid_checks
+    elif source == 'metrics':
+        get_valid = get_metric_sources
+    elif source == 'testable':
+        get_valid = get_testable_checks
+    else:
+        get_valid = get_valid_integrations
+
+    if checks is None or checks.lower() == 'all':
+        choice = sorted(get_valid())
+    elif checks.lower() == 'changed':
+        choice = sorted(get_changed_checks() & get_valid())
```

Comment:
From reading the underlying util being used, doesn't this only return checks that have changed python code? If someone just changed a README.md or manifest.json would that still be a "changed" check?

Reply:
Good catch, very true this will limit to only python code.  Will add new helper function.